docs/3.0.0-beta.0/concepts/concepts.md

```diff
@@ -0,0 +1,317 @@
+# Concepts
+
+- Table of contents
+  - [Files structure](#files-structure)
+  - [Controllers](#controllers)
+  - [Filters](#filters)
+  - [Models](#models)
+    - [Attributes](#attributes)
+    - [Relations](#relations)
+      - [Many-to-many](#many-to-many)
+      - [One-to-many](#one-to-many)
+      - [One-to-one](#one-to-one)
+      - [One-way](#one-way)
+    - [Lifecycle callbacks](#lifecycle-callbacks)
+  - [Internationalization](#internationalization-and-localization)
+  - [Plugin](#plugin)
+  - [Policies](#policies)
+    - [Global policies](#global-policies)
+    - [Scoped policies](#scoped-policies)
+    - [Plugin policies](#plugin-policies)
+  - [Public assets](#public-assets)
+  - [Requests](#requests)
+  - [Responses](#responses)
+  - [Routing](#routing)
+  - [Services](#services)
+
+---
+
+## Files structure
```

Comment:
Missing some structure from the beta.

I might be blind but not seeing extensions nor exports